From 593a9ad3defea7bc65595a40ff04b550b48278b5 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Thu, 29 Oct 2026 09:06:42 +0100
Subject: [PATCH 1/2] Update the upstream signing key

Update dickey@invisible-island.net-rsa3072.asc to the current copy at
https://invisible-island.net/public/dickey@invisible-island.net-rsa3072.asc,
the URL the spec names for it (same certificate(s)).

The current key is needed to verify the release signature with
openpgpverify (sqv), which checks that the key was valid when the
release was signed.

Assisted-by: Claude Opus 5.5
---
 dickey@invisible-island.net-rsa3072.asc | 138 ++++++++++++------------
 1 file changed, 69 insertions(+), 69 deletions(-)

diff --git a/dickey@invisible-island.net-rsa3072.asc b/dickey@invisible-island.net-rsa3072.asc
index a44002a..68f5d5d 100644
--- a/dickey@invisible-island.net-rsa3072.asc
+++ b/dickey@invisible-island.net-rsa3072.asc
@@ -8,64 +8,64 @@ j5QQK8s1xItCovjy69UNRDV+bvtb9YOXTkv1tDgqPCBGidOr9oznN91S5hbUjoUu
 rZNwlZsQZ7GGz5MXt2I4aL6sPyvKSj7IdVa3R2g+z8QLxvQWCN6V2FOcrziZ/EK+
 Rpnaw3C7+siz+lWGxVo1CJBks+V0lviA9Sd9MLJBJoxgJSao1YLkBSE9UXTBx1Va
 RU1buTg/0+2BrxEdX+9qIb4Qp/kaa33+Jw1IDauO6TQALilw2D1idkLZwlzMcfH0
-Mvlvd6DVWkVZ29EAEQEAAbRFVGhvbWFzIEUuIERpY2tleSAoc2VsZi1zaWduZWQg
-dy9vIFNIQTEpIDxkaWNrZXlAaW52aXNpYmxlLWlzbGFuZC5uZXQ+iQHUBBMBCgA+
-AhsDBQsJCAcCBhUKCQgLAgQWAgMBAh4BAheAFiEEGYgtkt2kxADCLA1WzCr0RyFn
-vgMFAmV2RIUFCQho4R0ACgkQzCr0RyFnvgM4sgwAo8RkSxARoU3igEm4tkhYBbRF
-Foz4ssT1RLFo/MDfSF735apiXrUKAItyF6YhDCg+C9qSeFqSBESIgtsXlF8aJXwn
-Xz1RdhYBzAyHi144JnuwGr3G6pAvka5vvncdyAp6k8hZI/FGKQhsYpf7FEgcHSn1
-9roMKGuJmjcTQfRnxl3zUaAWQAhC0AW6iWYdXdbFUe6dNqazSxlJozJdpUpnEwyq
-LtYNahZWjXAMSrqNwBH+J3BiaNorkypLvtUA/n0sxqgS4mun3ObnatFn0Z/uBtZ6
-mcdMf8JYgkOdQuwirNlwBRfvdip/40NntyxRZQ5mHtUqjx1iOzMFEsFG2xz2+7zw
-YmplY2YpS4aQTPyne6S+M2TmriMOZWdXm3UKLh3GcRAMPPFJ/XTvtDrDyhpcbwou
-mYjr+0tzNnWYEfsLsRrbDdf3O2FvkGiZNbGt3dQNIsTNNSJKjw+OmQDFMo04vutu
-Ey9qC5J99OS3Bk87MX4MnH+S7+XDGL7eIOthGBJtiQIcBBABCgAGBQJhzfBDAAoJ
-EO1sijiDR2RVNVYP/2Mx0ltBnKjY9nNOKOmZ7PJTz4Wz11dk4SUbn/bhXg45URqx
-x81+0UscCXHIGU21eVO6ObdTCGf57zxpkDJh18AZufYK6rT1Y9aytGPcWBPHMPWq
-+gvS38amjuYdtxz9YjeM6tolaZRaFuST3gMfVqdeL0pZpsLgJuDoNrl50ioAbbAX
-2Cs8u5ymlCyCt0nAt1Z9bxnSWXvd+pEYhMJMPlLp6SkWRKgcJlRJ8LR4vWVXPGUM
-T2dS+PlZDakDgYrizW68hu67c3CfNyGr1Jtx5G9j9YMbfCQUmbMwR1IdvXl+UBPv
-akFQsJHGhwi2sRsxlP9vpInKmCXziTtPYEp1qsiX6FgI4r60ceWVAOYLb0wl4+bb
-E6SqlmXsLNKrl+y9FAOqChp/r2KWZdQKMognbcZwK98adxbyQCCzYYNTsZVXs+N1
-orVQ/wLZLFryIzXZhsD2WFtGv86LXQ6T9wlTb8MPiR5cIpiHZSyzHamTd9Oi6acK
-60gx9qcQoybAVXPmBfCr+M4sX0cp0TnRiEQVR+CCHe+vr2oDp/SbE+uHSWzH57/t
-O98oDrdz1frSWlwiB1qdUYFyv3BAjl0hO2uzNiRJPt2Zpu8YAMOsz777y89lHwKQ
-yA81xh2ewKe+r4sk7YG2b0wU81uYIrMpYwI8wAHHzwxUQbAIpAzQiXJOwmmEiQIz
-BBABCgAdFiEEfDq5z9IwvTDdAJxZHnCRsfFKZKIFAmHN6I8ACgkQHnCRsfFKZKIU
-BQ/9FdoYLt7ZBw3GkfdomVCBs1K1BFXcVPRR7E1+PLTZVlXzkZ78OnCLV7EpharH
-8PBxyVZnzHBL38+5TKbQePgJujP4A/4zDgTlSACPoOT7sRHZ4dFPatVgGKJDW/JO
-6IvjQ8AazNuD6HjOKRkjBfodHrn5syKGTkgMeaiVDn1goZiDEc7vF64Tg4Jl3Bz2
-enh7d9T0ouu64uDz1/IMI9IO0IE7L0Y0rHEYUR7Zz1u0YFUT7RJsdS7SfMyaC/4c
-k/8u7z9uG+dK0D5KiToMvRoyAJgY76vCHEo1SEDCrTNtKrYTCl8KwNCojexcDXsl
-ujlh2clAs2pljqVlMxB/eEAcY/N2BEWi+xjUfUerfwRzzswXyQbQo/BXUc/mTIAP
-QA9jhRBmKrn7mGx7IWMuEWEC+kwVz1ecRM31kxIKoWTnb0i1Mmvl+MbzVuJawhFM
-+OWIg7DiuEMWTR8oPp3qEdr6o3W9j5dzHl2JhMKmBA0tTuRskhMEvoAfBR7AYM6i
-/Ynm/breqb0H7iGBhUSMpWbxXlgnx2ROk3TMA0lz1WIjBs8IXi83ZTGwyeEVLf7I
-m0iU+0pkyJ/ex5e1pbOKl16hoOKWwBI0Zfj8xnPTyXdAOA1YvCKLrzxEoSgkm/Bl
-wZamPk4YydbNLeTLR7QsHykiYxD2aegI8WDslmCj71PCbGyJAjMEEAEKAB0WIQRv
-t3u4nEHIyNLStZoZsmeAIQMynwUCYc44cAAKCRAZsmeAIQMyn8KyEACzIGuptlSC
-qDzIQcGjdYS0Z67ncMeQXw0WjRhXFPejc5phae8ifxdYco3yXgVTWIhc4Dhf6iBY
-H8X5Ah/Jb0ijR6qvp3tOHVj54hI2+0OCIkQJo/BypHstWmisvk19U8PzwzSM9R14
-u66RWxrv1a7is33ofx/kfACRIt2y5x4WCgxI8TJlehPz/HKYNRkmsbJDES2SBEGg
-8hNsko2HwFGwkYrJFqY5oaODFa542MdiV/3OaAQI64SaGMo3/j3TniBtzP0I7b43
-buFhxAIlAByxqRHlksOKNdH3Y4H5zNZ65L6ywVlGzlA+YodJKeWcgNBp46Ttu/fl
-fEFw/IKYIb9zOKUl5RbRRxhO4JSrttO2ROiKM0I0Fbo6B2hndN7kaQ7pIb+AfU1o
-K/Ro3Yr4gbhk+slotUsBlfhH4A4wfi41LrH27e+QtrMlxi+mguilmY00S55v9Om4
-sS5eWc/X00inqOc9J7DOuZ3wjp3AwFSILprjZkoV+CbgkvdXDHoY2G2gfzY5NMgl
-xxk7otW9bAe2hUxr5FhB9NaJ4ug4mIbkRrZc/tEPzIIqTdRRNRjJ7cB68D/JApgk
-140u2xsMScaSvo4UHeZE6NhWteRCKuQSImc6j+TGfWypwkKsR8GomRKvvAm5XMfc
-+7j5Om39kjYGcMsOhAChXT9X63Zucj8x/LQxVGhvbWFzIEUuIERpY2tleSAodXNl
-IGZvciBlbWFpbCkgPGRpY2tleUBoaXMuY29tPokB1AQTAQoAPgIbAwULCQgHAwUV
-CgkICwUWAgMBAAIeAQIXgBYhBBmILZLdpMQAwiwNVswq9EchZ74DBQJldkSJBQkI
-aOEdAAoJEMwq9EchZ74DvkIL/jIepvH9VeRu7fq3C41I3+tZdN0yFOAxZCwfmpUK
-im/U4++7uV3oDhGdAUZ0hKNRLld12bKTqItZKBhXkal5//UXZfemlGud1o3LToge
-ay3tFSCpO4XPuaigkSgZFMwblEgDVv9Kr6LgdwVnXNsdMCzctAygAZ/oDn9zj6Kc
-uf7wDM71GqMCRuY8olxRlo6fsqORf+i4GNf70Fc/eZo0zKYhH75wazv0goNGqjX+
-Ab/BS2kv1H7npMjz6F8kJ9jgzDCIbIY/VS7h6r0pKmLsvpPXkrZ0YR8SALIrm6RO
-/61WVRkLHsG25NlXK6g8ytRgITawqullPgCUve/wt4XQ+YRWDRDa9moz4muOIA/g
-QbWUtZZRVoLWugz28ffNIRFEjamarpV9xddw4jZvXzf1XzIDBZZH0VGiSUMpkjHh
-7cbx9miJkJHIb5WC7NPNy5sjm9XB/beRHrXNT5NA80FzmQ0uqA5rKa+7zu2VG9QB
-Y+TeoH4b9Ft+axAshCIQ6UpZF7kBjQRhBIZoAQwAvJYrnT5LdI2wXlLzr1bp9q4T
+Mvlvd6DVWkVZ29EAEQEAAbQxVGhvbWFzIEUuIERpY2tleSAodXNlIGZvciBlbWFp
+bCkgPGRpY2tleUBoaXMuY29tPokB1AQTAQoAPgIbAwULCQgHAwUVCgkICwUWAgMB
+AAIeAQIXgBYhBBmILZLdpMQAwiwNVswq9EchZ74DBQJpbD3GBQkObjJeAAoJEMwq
+9EchZ74D4B4L/A9ctDdLHv0rr3jRukbMtwIdnQSQvwsRRu8cyaGwko1Hp3tTCIov
+elradmGVst9yH+rp7/caJQFDE5IZBCh3YqdkmPbKq8XuwUX7T91B9PVpWDPF+l3A
+6UMk8ll3YORlyRgHXwDT4ZsY5yvuPJkropEV3TSbnYJi6X3gtB06mGZsDDzmNBOd
+6KuetO/0kCiO1o8X2PXL5iq3wmvz5tUKIBiNgDUKZFkVpz/HPvCVmcXLgXR040N2
+vfovzi6papl7JT12D8+yFs15SXONUaF+zKPxvHSy9rAyAqCSAYjzhecagMwMYvKb
+oBwr0NIzKOJruCFe7NGOOjOWgMNAtWsWTruSRc0mNEqmAOiyouJr9O2bpoyY6nf/
+shToho3d5qXXYWJTyAclJ6cIPmxOZdwTb6eUAqGtfjinu4PqbMNFVq7BqgYnJVWB
+VB/p9ODYJxJTqXpvuf1InR5BTqVNTkTRBjkQPxbNHFyxr/nZj8kZVKY3Ycq95CQS
+wSl+iCs8A3FuJ7RFVGhvbWFzIEUuIERpY2tleSAoc2VsZi1zaWduZWQgdy9vIFNI
+QTEpIDxkaWNrZXlAaW52aXNpYmxlLWlzbGFuZC5uZXQ+iQHUBBMBCgA+AhsDBQsJ
+CAcCBhUKCQgLAgQWAgMBAh4BAheAFiEEGYgtkt2kxADCLA1WzCr0RyFnvgMFAmls
+PcYFCQ5uMl4ACgkQzCr0RyFnvgN8YQwA4aZVz+ehuL1Kq+fS3kqYt8nfLB6f5OX4
+9kL+kUtPBug6xhfog8RinrFq7JmSsZ+id/IZc79OYG3gOxs8HvdIEZe41wL0yfzi
+pjRQuioy9Mh0QFtZFzBQUhbFhRBB3DwJIs2qoJYXuZT03RA5Ld61yzlNwzYW+l/G
+e8lSTvYdks8RPkY47ZAzbhfxFrzyV6oUObPlJEYMPz3kyq2pKJlh487jFKN7f2qe
+SgTTFz4egVdexJSnQxlCXrXkX91exLKNdp9PaYhODF6Vlypx5n7S0WLJl4MhpBDm
+dWglaIEOBqXDRV7PaIKnyMBL26TX/67wcwT1PfNBO3S4KE1rfQZDck80JvIKq160
+6TvPxayId860xePbq+I9MHUXzbwduf8a4gR68oriv+N5GOk4CYltEB8r1O1iN0Lq
+LtbA3KJ385ipx3X12cbhdobnLEQ3zXMC5BdNjcRKpc+NMWZOnxW4s5p3FFraGoIr
+8hb2yW+ZF82mdACc6Pc1B91nvMnwAYR+iQIcBBABCgAGBQJhzfBDAAoJEO1sijiD
+R2RVNVYP/2Mx0ltBnKjY9nNOKOmZ7PJTz4Wz11dk4SUbn/bhXg45URqxx81+0Usc
+CXHIGU21eVO6ObdTCGf57zxpkDJh18AZufYK6rT1Y9aytGPcWBPHMPWq+gvS38am
+juYdtxz9YjeM6tolaZRaFuST3gMfVqdeL0pZpsLgJuDoNrl50ioAbbAX2Cs8u5ym
+lCyCt0nAt1Z9bxnSWXvd+pEYhMJMPlLp6SkWRKgcJlRJ8LR4vWVXPGUMT2dS+PlZ
+DakDgYrizW68hu67c3CfNyGr1Jtx5G9j9YMbfCQUmbMwR1IdvXl+UBPvakFQsJHG
+hwi2sRsxlP9vpInKmCXziTtPYEp1qsiX6FgI4r60ceWVAOYLb0wl4+bbE6SqlmXs
+LNKrl+y9FAOqChp/r2KWZdQKMognbcZwK98adxbyQCCzYYNTsZVXs+N1orVQ/wLZ
+LFryIzXZhsD2WFtGv86LXQ6T9wlTb8MPiR5cIpiHZSyzHamTd9Oi6acK60gx9qcQ
+oybAVXPmBfCr+M4sX0cp0TnRiEQVR+CCHe+vr2oDp/SbE+uHSWzH57/tO98oDrdz
+1frSWlwiB1qdUYFyv3BAjl0hO2uzNiRJPt2Zpu8YAMOsz777y89lHwKQyA81xh2e
+wKe+r4sk7YG2b0wU81uYIrMpYwI8wAHHzwxUQbAIpAzQiXJOwmmEiQIzBBABCgAd
+FiEEfDq5z9IwvTDdAJxZHnCRsfFKZKIFAmHN6I8ACgkQHnCRsfFKZKIUBQ/9FdoY
+Lt7ZBw3GkfdomVCBs1K1BFXcVPRR7E1+PLTZVlXzkZ78OnCLV7EpharH8PBxyVZn
+zHBL38+5TKbQePgJujP4A/4zDgTlSACPoOT7sRHZ4dFPatVgGKJDW/JO6IvjQ8Aa
+zNuD6HjOKRkjBfodHrn5syKGTkgMeaiVDn1goZiDEc7vF64Tg4Jl3Bz2enh7d9T0
+ouu64uDz1/IMI9IO0IE7L0Y0rHEYUR7Zz1u0YFUT7RJsdS7SfMyaC/4ck/8u7z9u
+G+dK0D5KiToMvRoyAJgY76vCHEo1SEDCrTNtKrYTCl8KwNCojexcDXslujlh2clA
+s2pljqVlMxB/eEAcY/N2BEWi+xjUfUerfwRzzswXyQbQo/BXUc/mTIAPQA9jhRBm
+Krn7mGx7IWMuEWEC+kwVz1ecRM31kxIKoWTnb0i1Mmvl+MbzVuJawhFM+OWIg7Di
+uEMWTR8oPp3qEdr6o3W9j5dzHl2JhMKmBA0tTuRskhMEvoAfBR7AYM6i/Ynm/bre
+qb0H7iGBhUSMpWbxXlgnx2ROk3TMA0lz1WIjBs8IXi83ZTGwyeEVLf7Im0iU+0pk
+yJ/ex5e1pbOKl16hoOKWwBI0Zfj8xnPTyXdAOA1YvCKLrzxEoSgkm/BlwZamPk4Y
+ydbNLeTLR7QsHykiYxD2aegI8WDslmCj71PCbGyJAjMEEAEKAB0WIQRvt3u4nEHI
+yNLStZoZsmeAIQMynwUCYc44cAAKCRAZsmeAIQMyn8KyEACzIGuptlSCqDzIQcGj
+dYS0Z67ncMeQXw0WjRhXFPejc5phae8ifxdYco3yXgVTWIhc4Dhf6iBYH8X5Ah/J
+b0ijR6qvp3tOHVj54hI2+0OCIkQJo/BypHstWmisvk19U8PzwzSM9R14u66RWxrv
+1a7is33ofx/kfACRIt2y5x4WCgxI8TJlehPz/HKYNRkmsbJDES2SBEGg8hNsko2H
+wFGwkYrJFqY5oaODFa542MdiV/3OaAQI64SaGMo3/j3TniBtzP0I7b43buFhxAIl
+AByxqRHlksOKNdH3Y4H5zNZ65L6ywVlGzlA+YodJKeWcgNBp46Ttu/flfEFw/IKY
+Ib9zOKUl5RbRRxhO4JSrttO2ROiKM0I0Fbo6B2hndN7kaQ7pIb+AfU1oK/Ro3Yr4
+gbhk+slotUsBlfhH4A4wfi41LrH27e+QtrMlxi+mguilmY00S55v9Om4sS5eWc/X
+00inqOc9J7DOuZ3wjp3AwFSILprjZkoV+CbgkvdXDHoY2G2gfzY5NMglxxk7otW9
+bAe2hUxr5FhB9NaJ4ug4mIbkRrZc/tEPzIIqTdRRNRjJ7cB68D/JApgk140u2xsM
+ScaSvo4UHeZE6NhWteRCKuQSImc6j+TGfWypwkKsR8GomRKvvAm5XMfc+7j5Om39
+kjYGcMsOhAChXT9X63Zucj8x/LkBjQRhBIZoAQwAvJYrnT5LdI2wXlLzr1bp9q4T
 0n+xnuIuMTTti4zN6OoctsbKiqouUIVJfb2rH+Dm3GIfK4dlscA1iXC4XEEk0bUQ
 Ln6o3VbXph/YZ3FLJ9vB0bsPxO7MmEG0QXV1RNQU1hbum8oXinK2e7lfLaQeRU2U
 n2nC2FVHNRmUst0NSdOGlalmVq3vX5DfRC1SafbDPerP9tGetMawskElvXfhTPzc
@@ -74,15 +74,15 @@ wdAuzu+0K60xHKLXFxotBm0Q+hQs+dfJXMIkxVPmL4aQ/UEnpFJboxdpHhxTHWox
 cBgc7DKDPITO1LLyA19vCkXM/Z++dzbCF9h7TturCbt2fZAY9UhXVE1dtLMWypmx
 eiNHrVaXO8eLXNWwT1xBSngWaYfmbt5s8XL7ZWWK2jrqkmRa9ODMfuJ4yCnh5JcK
 /xluaGmRtSSDx9B3Xcz0TlKcGFxJQiUYhKtTmzXrABEBAAGJAbwEGAEKACYCGwwW
-IQQZiC2S3aTEAMIsDVbMKvRHIWe+AwUCZXZElAUJCGjhLAAKCRDMKvRHIWe+A6h8
-DADvjWLQ+r1y/XMcXFhtJT+hLWY3TtVQAcxqCa/VkXR1A0Tc5FbxU5nK6ASvDySs
-kdP2r/xHnFapOmzUA5HJJhHrXrz8cWo3umrIob4gWVegkQzWk3Ypcr9prjEF8Rvj
-EkphR5vzbSnigQykGYx4vfYvPCC3PH8BlrI0GJgWnMActbHEmLw5ZEkX3QNrOk22
-uV5qBjfN5nq4TCD8CpUBQLUsGl18WS2S68bQdCqrQD32fmcByDPJDCnzVElh9UFQ
-1DP8XE1T8FZMZ9DLhUs6F7CY6MINmQAKzqAyfTgcG/bEC+9mDwiHjEp+HdBGariz
-oO4aMpnpjqUa9KU/pQBQoUnCC2Tv17fUyu6qIma3TZ6gkcVwY4Jo0y2jcLgyy4wY
-Y54otUE9idmqyY+M+YAOjUDGHcGn9httYKyXCYasmKHpLu8aQ95hUdA8rRJE9+N6
-Bqla1Ei4VIsxsXXbE0R7YGF9LB0WM+aOXioCQEZRCOXgcE38Ki4L59v7CQgadUIT
-b/Q=
-=WuBN
+IQQZiC2S3aTEAMIsDVbMKvRHIWe+AwUCaWw99wUJDm4yjwAKCRDMKvRHIWe+A32d
+DAC/0ucqxiadAGRhPgyxUoRzZua7fDNoC40/5dXrPLY702HZVaThIvFi/GJwG7hn
+G8X+tDzK3fzjCS+2hMHd0a4flqyrPNTNhpBnDbyTpUhlE5e9cCIqnYrClwq6l5Cv
++LPQacjoarxLlvAsqybY/6UJYhVAgBJHNzp7Q16JmzIoBtg4/zS8mY/azmRTbiqt
+z7Uu+whkJf0wWS7fdJAho1ke1PNTJ+uX7M+BNAKgDhIorthx9N3Sgh5X4i2lJVeB
+i7xUlJ2/dBJKS1F7IxnALM2r8Nkqbupt17dqrKVLbmFanRbxxItM38jDHOQhsZID
+IpqYjFs7th/8p05GZAThj52ujKlb6kEzmZ/dqMKQ9N1ZtAtimMdsMRP52U9ZtQyn
+B+Pa1skFAb8WFtBvIlVghPM5UwA2SBtUBUm0ROzVGMm+roFfngDpowQ8NH9o+ldB
+s8IVxqi6wKpIRtjmDp8CzjtMGtiQjLTz//Ttgh62mkM3RfD/opCgz8lB0a4tXxK9
+YbI=
+=p0/w
 -----END PGP PUBLIC KEY BLOCK-----
-- 
2.56.0


